edge/pkg/edgehub/certificate/certmanager_test.go

```diff
@@ -0,0 +1,65 @@
+package certificate
```

Comment:
Copyright should be added in a new file, and the below files also should add.

Reply:
done. Added copy right headers to all the newly added and modified files.